.github/workflows/ci.yaml

```
@@ -30,4 +31,13 @@ jobs:
         run: cargo clippy --all-targets
 
       - name: Fmt
-        run: cargo fmt --all -- --check
\ No newline at end of file
+        run: cargo fmt --all -- --check
+      - name: Fmt
+        run: cargo fmt --all -- --check
+
+      - name: Doc
+        run: |
+          cargo doc --workspace --all-features
+          cd macros
+          cargo doc --workspace --all-features
```

Remove unnecessary `--workspace` and use `--no-deps` to speed up the CI.

```suggestion
          cargo doc --all-features --no-deps
          cd macros
          cargo doc --all-features --no-deps
```